app/server/models/transfer_account.py

```diff
@@ -29,6 +29,12 @@ class TransferAccount(OneOrgBase, ModelBase, SoftDelete):
 
     name            = db.Column(db.String())
     _balance_wei    = db.Column(db.Numeric(27), default=0)
```

Comment:
this can be nuked

Reply:
`_balance_wei` is still used though, it's just set differently